parm/config/config.resources.emc.dyn

```diff
@@ -160,7 +160,7 @@ elif [ $step = "analcalc" ]; then
 elif [ $step = "analdiag" ]; then
 
     export wtime_analdiag="00:10:00"
-    export npe_analdiag=72
+    export npe_analdiag=115		# Should be at least twice npe_ediag
```

Comment:
Setting this to twice npe_ediag is sufficient.   Currently npe_ediag=48.   Therefore, set npe_analdiag=96..   If concerned that this is too low, use the operational WCOSS_D values of npe_ediag=56 and npe_analdiag=112.  Similar comment applies to `config.resources.nco.static`

Reply:
Noted, thanks @RussTreadon-NOAA !